backend/RoomPlanner/room-planner/src/main/resources/db/migration/patch_1.4_add_authentication_tables.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE oauth_access_token (
 );
 
 CREATE TABLE oauth_client_details (
-    client_id character varying(256) NOT NULL,
+    client_id character varying(256) primary key NOT NULL,
```

Comment:
You should create a new patch not editing an existing one

Reply:
I need that to test my query with H2